include/zephyr/logging/log_backend_adsp_mtrace.h

```diff
@@ -9,6 +9,20 @@
 
 #include <stdint.h>
 #include <stddef.h>
+#include <adsp_types.h>
+
+#define MTRACE_CORE		0
+
+#ifndef ADSP_PAGE_ALIGNMENT_SIZE
+#define ADSP_PAGE_ALIGNMENT_SIZE 0x1000
+#endif
+
+#ifndef PAGE_SIZE
+#define PAGE_SIZE ADSP_PAGE_ALIGNMENT_SIZE
```

Comment:
I am not sure if this header is good place to define PAGE_SIZE. There is another define ADSP_DW_PAGE_SIZE.

Reply:
Hi @jxstelter, there was ADSP_DW_PAGE_SIZE in adsp_debug_window.h. I removed that file. But other than that you're right. I followed the example and added page size define to each module by analogy. Not the best solution. I'm thinking about Kconfig option now. Do you have any better suggestions where to put it? I personally haven't found proper existing header file, didn't want to create a separate one just for this define and still want those modules to be as independent from each other as possible to provide some kind of encapsulation.

Reply:
What about CONFIG_MM_DRV_PAGE_SIZE?

Reply:
CONFIG_MM_DRV_PAGE_SIZE works fine for most cases, but I believe that log backend mtrace in platforms other than ACE  is supposed to work without mm driver. Anyway I removed PAGE_SIZE from the header file.